modules/persistent-storage-vsphere-formatting.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-formatting-volumes_{context}"]
+= Formatting VMware vSphere volumes
+
+Before {product-title} mounts the volume and passes it to a container, it checks that the volume contains a file system as specified by the `fsType` parameter in the PersistentVolume (PV) definition. If the device is not formatted with the file system, all data from the device is erased, and the device is automatically formatted with the given file system.
```

Comment:
s/as specified/that is specified
s/parameter/parameter value
s/given/specified